.github/workflows/scripts/check_for_redirects.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  /**
+   * Add redirects-needed label if count of deleted files is greater than 0
+   *
+   * @param {Object} obj.artifactName - Name of artifiact file to check
+   */
+  addRedirectsNeededLabel: async ({ github, context, fs, artifactName }) => {
+    const {
+      payload: {
+        repository: {
+          owner: { login: ownerLogin },
+          name: repoName
+        }
+      }
+    } = context;
+
+    const artifactContents = fs
+      .readFileSync(`./${artifactName}.txt`)
+      .toString();
+
+    const [prNumberStr, numOfDeletedFilesStr] = artifactContents
+      .split('\n')
+      .map((str) => str.trim());
+
+    console.log('PR number that triggered workflow:', prNumberStr);
+    console.log('Number of deleted files:', numOfDeletedFilesStr);
+
+    if (
```

Comment:
Can we add an `else` statement that exits 1 with an error message?

Reply:
Using `core.setFailed` so that it exits out of the script.

Reference: 
- https://github.com/actions/toolkit/tree/main/packages/core#exit-codes